### DISCUSSION TITLE
Including dosing details into PK.results$result Table

### DISCUSSION THREAD

**Discussion by Gong-Chen** [humanpred/pknca#230] [Q&A] [answered]
I am new to PKNCA and got a question… Is there a way to include dosing details (e.g. dose level, dose duration and route) as additional rows into PK.results$result Table at the point of executing pk.nca? I tried to set Intervals and add.interval.col but didn’t have any luck. Thanks

**Comment by billdenney** [comment, marked as answer]
The way that I typically do it is to include it as a grouping variable.  Then, it is retained as a column (though not a row).

**Reply by Gong-Chen** [reply]
Thanks Bill.
